lib/Server.js

```diff
@@ -381,25 +381,22 @@ class Server {
                 }
               }
 
-              const bypass = typeof proxyConfig.bypass === 'function';
-              const bypassRawValue = bypass
+              // - Check if we have a bypass function defined
+              // - In case the bypass function is defined we'll retrieve the
+              // bypassUrl from it otherwise byPassUrl would be null
+              const isByPassFuncDefined =
+                typeof proxyConfig.bypass === 'function';
+              const bypassUrl = isByPassFuncDefined
```

Comment:
`isByPassFuncDefined` wasn't too confusing before, but no harm having it. The new code comments explain it :)